lang/en/tool_cloudmetrics.php

```diff
@@ -80,6 +80,17 @@
 $string['different_freq'] = 'Caution - frequency has been changed, new data will complete currently present but no data will be added between
 {$a->backfilledfrom} and {$a->backfilledto}.';
 $string['period_select'] = 'Select period to retrieve data from: ';
+$string['period_select_delete'] = 'Select metric to reset or delete: ';
+$string['reset'] = 'Reset';
+$string['delete'] = 'Delete';
+$string['period_delete'] = 'Are you sure you want to delete data from {$a->startdate} to {$a->enddate} for {$a->metric} metric ?';
+$string['period_reset'] = 'Are you sure you want to completely erase data for {$a} metric ?';
```

Comment:
period_delete and period_reset aren't currently used. An extra confirmation with these strings seems like it would be useful though.